daemon/containerd/image.go

```diff
@@ -352,3 +360,25 @@ func imageFamiliarName(img containerdimages.Image) string {
 	}
 	return img.Name
 }
+
+// getImageLabelByDigest will return the value of the label for images
+// targeting the specified digest.
+// If images have different values, an errdefs.Conflict error will be returned.
+func (i *ImageService) getImageLabelByDigest(ctx context.Context, target digest.Digest, labelKey string) (string, error) {
+	imgs, err := i.client.ImageService().List(ctx, "target.digest=="+target.String() + ",label." + labelKey)
```

Comment:
DOH! looks like there's a linting failure (probably spaces around the `+` that it's failing on);

```
INFO [runner] linters took 1m53.561907265s with stages: goanalysis_metalinter: 1m53.496004751s 
daemon/containerd/image.go:368: File is not `goimports`-ed (goimports)
	imgs, err := i.client.ImageService().List(ctx, "target.digest=="+target.String() + ",label." + labelKey)
```

Reply:
Yep, just fixed!